.travis/script.sh

```diff
@@ -4,9 +4,9 @@ set -x
 # If this is a master build then deploy at the end
 if [ "$TRAVIS_BRANCH" = 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
   mvn -Dstyle.color=always -B -U -P sign --settings travis-settings.xml deploy &&
-  # TODO - after testing this, make it only publish on a release build
-  mvn site site:stage post-site scm-publish:publish-scm;
+  mvn site site:stage post-site;
 # Otherwise just build and test
 else
-  mvn -Dstyle.color=always -B -U --settings travis-settings.xml verify;
-fi
\ No newline at end of file
+  mvn -Dstyle.color=always -B -U --settings travis-settings.xml verify &&
+  mvn site site:stage post-site;
```

Comment:
running site here just to make sure people can't accidentally break the site generation in a PR